migrations/20240326144400_users.sql

```diff
@@ -0,0 +1,14 @@
+-- +goose Up
+-- +goose StatementBegin
+ALTER TABLE users
+ADD hash_password VARCHAR(255) NOT NULL,
+ADD email VARCHAR(255) NOT NULL
```

Comment:
Now that you added a field for the password hash, it may be returned in the response body from a GET request to the `/users` endpoint
Make sure you leave them out of that request since it's sensitive information. You can do so by applying a `qm.Select` to the models query:

```
// getUsersHandler
users, err := models.Users(qm.Select("id", "name", "<any other fields you want>")).All(c, db)
```

Reply:
You also need to make sure it's not being returned anywhere else - it should just be used for auth